hardware/README.md

```diff
@@ -0,0 +1,35 @@
+# ssis:bit hardware
+
+The foundation of the ssis:bit is a [LilyGo T8 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) board with [CircuitPython](https://circuitpython.org/). It has already many key features out of the box:
+
+- ESP32-S2 with 240 MHz CPU, 4GB Flash, 8GB PSRAM
+- 1.14" ST7789 Display 240x135
```

Comment:
In REPL it is actually 37x9 characters (333). Less than the 2K from 80x25 of old terminals, but usable nonetheless.